packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -270,6 +270,33 @@ export const GridMixin = (superClass) =>
         reorderElements: true,
       });
 
+      // Chromium based browsers cannot properly generate drag images for elements
+      // that have children with massive heights. This workaround prevents crashes
+      // and performance issues by excluding the items from the drag image.
+      // https://github.com/vaadin/web-components/issues/7985
+      document.addEventListener(
```

Comment:
The event listener should be added on connectedCallback and removed on disconnectedCallback to avoid a memory leak.

Reply:
Done.